src/checkdata.py

```diff
@@ -21,6 +20,10 @@
 from src.data_utils.ProcessTrafficData import mergeSegment, LoadTrafficData, GenerateObsmat, \
     createMap, FilterTraffic
 
+sys.path.remove("/opt/ros/kinetic/lib/python2.7/dist-packages")
```

Comment:
Because I have ROS sourced at all times, this is necessary for searching the open-cv package at the right place. I know it is really ugly, but it the only way it works at my computer